src/elements/content-sidebar/__tests__/SidebarNav.test.js

```diff
@@ -138,6 +149,27 @@ describe('elements/content-sidebar/SidebarNav', () => {
         });
     });
 
+    test('should call focusBoxAISidebarPrompt when clicked on Box AI Tab', async () => {
+        render(
+            getSidebarNav({
+                features: {
+                    boxai: {
+                        sidebar: {
+                            showOnlyNavButton: false,
+                        },
+                    },
+                },
+                props: { hasBoxAI: true },
+            }),
+        );
+
+        const button = screen.getByTestId('sidebarboxai');
+
+        await userEvent.click(button);
+
+        expect(focusBoxAISidebarPromptMock).toHaveBeenCalled();
```

Comment:
nit: Can we ensure this is called once using `toHaveBeenCalledTimes(1)` and/or ensure the proper params are passed into the function being called